src/panels/meeting-times.js

```diff
@@ -72,45 +78,111 @@ var MeetingTimesPanel = React.createClass({
         else {
             var pageTimes = this.state.groups.slice(index, index + ROWS_PER_PAGE);
             var items = pageTimes.map(function(meeting) {
-                var location = meeting.meetingLoc ? meeting.meetingLoc : 'TBA';
-                var meeting_time = moment(meeting.meetingTime, 'h:mm A', true);
-                meeting_time = meeting_time.isValid() ? time.formatTime(meeting_time, true) : undefined;
-                var meeting_date = time.formatMeetingDate(meeting.meetingDay) || meeting.meetingDay;
-                var fulltime = (meeting_date && meeting_time) ?
-                               (meeting_date + ', ' + meeting_time) : 'TBA';
-                return <tr key={meeting.name}>
-                    <td>{meeting.name}</td>
-                    <td>{location}</td>
-                    <td>{fulltime}</td>
-                </tr>;
+                if (displayingMeetings) {
+                    var location = meeting.meetingLoc ? meeting.meetingLoc : 'TBA';
```

Comment:
Location, meeting_time, meeting_date, and fulltime can be calculated outside the if/else block

Reply:
I removed them from the else block since they weren't used there - I can move them and the chair stuff outside the if/else, but then they would be calculated where they're not needed.